docs/docs/get_started/kubernetes_tutorial.md

```diff
@@ -233,151 +252,152 @@ kubectl --namespace quick-start-backend-ns  apply -f pg-service.yml
+**Note:** _If you're using your own database server and it's not SSL-enabled,
+please see the [handler documentation](/docs/reference/handlers/postgres.html)
+for how to disable SSL in your Secretless configuration._
 
-Please ensure the environment variables are set to reflect your environment. For example, if you followed along in the last section and are using minikube for your local K8s cluster, you can run:
+If you followed along in the last section and are using minikube, you can run:
 
 ``` bash
-export REMOTE_DB_ADMIN_USER=quick_start_admin_user
-export REMOTE_DB_ADMIN_PASSWORD=quick_start_admin_password
+export SECURITY_ADMIN_USER=security_admin_user
+export SECURITY_ADMIN_PASSWORD=security_admin_password
 export REMOTE_DB_URL="$(minikube ip):30001"
 ```
 
-You will setup and configure the PostgreSQL storage backend by carrying the following steps:
-1. Create a dedicated application database (henceforth referred to by the environment variable `APPLICATION_DB_NAME`) within the PostgreSQL DBMS, then carry out the rest of the steps within its context
-2. Create the application table (i.e. pets)
-3. Create an application user with limited privileges: `SELECT` and `INSERT` on the application table
-4. Store the application user's credentials (held in the environment variables `APPLICATION_DB_USER` and `APPLICATION_DB_INITIAL_PASSWORD`) in in a secret store (for the purposes of this demo, we're using Kubernetes secrets).
+Next, we'll create the application database and user, and securely store the
+user's credentials:
+
+1. Create the application database
+1. Create the `pets` table in that database
+1. Create an application user with limited privileges: `SELECT` and `INSERT` on
+   the `pets` table
+1. Store these database **application-credentials** in Kubernetes secrets.
+
+So we can refer to them later, export the database name and
+application-credentials as environment variables:
 
-**Note:** You must set the value of and export the environment variables `APPLICATION_DB_NAME`, `APPLICATION_DB_USER` and `APPLICATION_DB_INITIAL_PASSWORD` before proceeding, e.g.
 ``` bash
 export APPLICATION_DB_NAME=quick_start_db
 export APPLICATION_DB_USER=app_user
 export APPLICATION_DB_INITIAL_PASSWORD=app_user_password
 ```
 
-To create the application database, application table, application user and grant the application user relevant privileges, run this command:
+Finally, to perform the 4 steps listed above, run:
 
 ```bash
-docker run \
-  --rm \
-  -i \
-  -e PGPASSWORD=${REMOTE_DB_ADMIN_PASSWORD} \
-  postgres:9.6 \
-    psql -U ${REMOTE_DB_ADMIN_USER} "postgres://${REMOTE_DB_URL}/postgres" \
+docker run --rm -i -e PGPASSWORD=${SECURITY_ADMIN_PASSWORD} postgres:9.6 \
+    psql -U ${SECURITY_ADMIN_USER} "postgres://${REMOTE_DB_URL}/postgres" \
     << EOSQL
-/* Create Application Database */
+
 CREATE DATABASE ${APPLICATION_DB_NAME};
 
-/* Connect to Application Database */
-\c ${APPLICATION_DB_NAME};
+\c ${APPLICATION_DB_NAME}; -- connect to it
```

Comment:
this doesn't seem to work for me - running it as is gives the error
```
\connect: invalid port number: "to"
```

Reply:
removing ` -- connect to it` got it working

Reply:
I changed back to `/* ... */` style comments above